src/main/js/helpers/translateUtil.js

```diff
@@ -0,0 +1,20 @@
+import constants from "./constants";
+
+/**
+ * Makes translation zero if panning is enabled
+ *
+ * @private
+ * @param {object} config - config object derived from input JSON
+ * @returns {undefined} transcall - holds the value of transition.
+ */
+const translatePan = (config) => {
+    let transcall;
```

Comment:
I like the idea but can we have this in a way where it is more functional rather than procedural in nature?
Lets say that I have a dictionary entry based on https://github.com/cerner/carbon-graphs/issues/41#issuecomment-510461080

```js
// In GraphConfig.js, GanttConfig.js etc.
const settingsDictionary = (config) =>
    isPanningModeEnabled(config)
        ? {
              shouldClamp: false,
              transition: {
                  duration: 0,
                  ease: "linear"
              }
          }
        : {
              shouldClamp: true,
              transition: {
                  duration: 250,
                  ease: "linear"
              }
          };

```
Now when you are using the transition, you can just do `d3Transition(settingsDictionary.transition)` instead. Change the `d3Transition` in `constants.js` function to:
```
const d3Transition = (d=250, e="linear") => (transition) => transition.duration(d).ease(e);
```

You wouldnt need this function neither `d3TransitionPan`.

Reply:
`// In GraphConfig.js, GanttConfig.js etc.`
Will it be worth to add same peace of code in all the config files, it will be a redundant piece of code.

Reply:
That's ok for now, we already have a lot of duplicate code that we might want to trim later on. 
As for this piece of code, it will be in 3 different places as opposed to considerable more amount of if-else checks.

Reply:
ok, I will do the above changes.

Reply:
Since transition is at many places like axis.js dateline.js etc we should have a commonplace for our logic this way it won't work.

Reply:
Let's have this function in a different js file in helpers?